src/pynwb/data/nwb.ecephys.yaml

```diff
@@ -335,6 +335,20 @@ groups:
   - name: location
     dtype: text
     doc: description of location of this electrode group
+  datasets:
```

Comment:
Shouldn't this be a link?

will you only ever want to link to a single ElectricalSeries?

Reply:
Yes, this should be a link